src/components/Title.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+import styles from './Title.scss';
+
+const Title = React.createClass({
+
+  render() {
+    return (
+      <div className={styles.container}>Kodeklubben</div>
```

Comment:
Blir det litt overkill å lage en komponent for å rendre ett ord? Kan dette (<div> + scss) flyttes inn i FrontPage?